ui/src/app/modules/environments/configuration/configuration.style.css

```diff
@@ -0,0 +1,17 @@
+>>> .config-entry li {
+  width: 47%;
+}
+
+>>> .config-entry li.config-del {
+  float: right;
+  padding: 0;
+  width: 4%;
+}
+
+>>> .config-entry li.config-name {
```

Comment:
No. I know it seems weird but it is so Angular will 'deep' copy the styles to dynamically added elements. Seems very VERY weird but that is what the interwebs told me to do